ui/src/main.mjs

```diff
@@ -232,8 +232,10 @@ fetch('_setup')
                 getProperty (property) {
                     const config = this.props ? this.props[property] : null // last known value for the config of this widget property
                     const state = this.state[property] // chec if there have been any dynamic updates to this property
+                    // check if the state is null or a string 'null'
+                    const isNull = typeof state === 'string' ? state === 'null' : state === null
```

Comment:
why would this be a string `null`?

Reply:
While discussing with @Steve-Mcl last week, he suggested that it would be beneficial if we could also set `null` by typing it directly into the widget's edit mode, similar to how we currently set `null` when changing the data type to `JSON`.

<img width="618" alt="Screenshot 2024-09-11 at 16 44 02" src="https://github.com/user-attachments/assets/b9b37713-07de-4b53-848d-0f27cb4786b0">

Reply:
@gayanSandamal, I've just checked with @Steve-Mcl and this is not correct. We should not support `"null"` as a string, it should be `null` as a `null`.

Reply:
Reverted the string `"null"` checking